index.bs

```diff
@@ -10313,18 +10491,44 @@ Note: An {{AudioWorkletGlobalScope}} is associated with a single
 for that context. This prevents data races from occurring in global
 scope code running within concurrent threads.
 
+<div class="addition proposed" id="c2456-5">
+<span class="marker">Proposed Addition
+	<a href="https://github.com/WebAudio/web-audio-api/issues/2456">Issue
+		2456</a>.
+</span>
+Add a MessagePort to the AudioWorkletGlobalScope
+<div class="amendment-buttons">
+</div>
+<del cite=#2456>
+<xmp class="idl extract" data-no-idl>
+	callback AudioWorkletProcessorConstructor = AudioWorkletProcessor (object options);
+
+	[Global=(Worklet, AudioWorklet), Exposed=AudioWorklet]
+	interface AudioWorkletGlobalScope : WorkletGlobalScope {
+	undefined registerProcessor (DOMString name,
+	AudioWorkletProcessorConstructor processorCtor);
+	readonly attribute unsigned long long currentFrame;
+	readonly attribute double currentTime;
+	readonly attribute float sampleRate;
+	};
+</xmp>
+</del>
+<ins cite=#2456>
 <pre class="idl">
 callback AudioWorkletProcessorConstructor = AudioWorkletProcessor (object options);
 
 [Global=(Worklet, AudioWorklet), Exposed=AudioWorklet]
 interface AudioWorkletGlobalScope : WorkletGlobalScope {
 	undefined registerProcessor (DOMString name,
-	                             AudioWorkletProcessorConstructor processorCtor);
+								 AudioWorkletProcessorConstructor processorCtor);
```

Comment:
Does this show up correctly after bikeshed processing?

Reply:
It does yeah.